src/views/EditorHome.vue

```diff
@@ -247,6 +248,15 @@ export default defineComponent({
           return false; // this is the same with event.preventDefault()
         },
       ],
+      [
+        "テキスト欄を複製",
+        () => {
+          if (activeAudioKey.value != undefined) {
+            duplicateAudioItem();
+          }
+          return false;
+        },
+      ],
```

Comment:
> 意図的にテキスト欄にフォーカスがある状態ではショートカットを打てなくしてあります。

理由に書いてくださっているCOMMANDを待つ方法がない点、まさしく仰る通りなんですよね･･･。
ここにこんな感じのFIXMEコメントを書くのはどうでしょう。

```suggestion
      [
        // FIXME: テキスト欄にフォーカスがある状態でも実行できるようにする
        // https://github.com/VOICEVOX/voicevox/pull/1096
        "テキスト欄を複製",
        () => {
          if (activeAudioKey.value != undefined) {
            duplicateAudioItem();
          }
          return false;
        },
      ],
```

Reply:
たしかに！suggest取り込ませていただきました 🙏